packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -60,25 +60,56 @@ const useIntentIconStyles = makeStyles({
   },
 });
 
+const useIntentIconStylesInverted = makeStyles({
+  success: {
+    color: tokens.colorPaletteGreenForegroundInverted,
+  },
+  error: {
+    color: tokens.colorPaletteRedForegroundInverted,
+  },
+  warning: {
+    color: tokens.colorPaletteYellowForegroundInverted,
+  },
+  info: {
+    color: tokens.colorNeutralForegroundInverted2,
+  },
+});
+
+const useActionButtonColorInverted = makeStyles({
+  action: {
+    color: tokens.colorBrandForegroundInverted,
+    '&:focus': { ...shorthands.outline('0') },
+    '&:focus-visible': {
+      zIndex: 1,
+      ...shorthands.outline('2px', 'solid', tokens.colorNeutralBackground5Pressed),
+      ...shorthands.borderRadius('4px'),
+      ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
+    },
```

Comment:
We don't use `:focus-visible` directly as this feature does not match our browser matrix. We have a special set of macro functions from `@fluentui/react-tabster`, one of them is `createCustomFocusIndicatorStyle`.

As I see you're overriding `Button`'s styles, I briefly checked and it seems that following override does the same:

```ts
    ...createCustomFocusIndicatorStyle(
      {
        ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
        outlineColor: tokens.colorNeutralBackground5Pressed,
      },
      { enableOutline: true },
    ),
```

Reply:
You will also need to add `@fluentui/react-tabster` to dependencies:

#### `packages/react-components/react-alert/package.json`
```diff
    "@fluentui/react-icons": "^2.0.175",
+   "@fluentui/react-tabster": "^9.3.2",
    "@fluentui/react-theme": "^9.1.3",
```